Neos.ContentRepository.Core/Classes/CommandHandler/CommandBus.php

```diff
@@ -35,9 +36,13 @@ public function handle(CommandInterface $command): EventsToPublish|\Generator
     {
         // multiple handlers must not handle the same command
         foreach ($this->handlers as $handler) {
-            if ($handler->canHandle($command)) {
-                return $handler->handle($command, $this->commandHandlingDependencies);
+            if (!$handler->canHandle($command)) {
+                continue;
             }
+            foreach ($this->commandHooks as $commandHook) {
+                $command = $commandHook->onBeforeHandle($command);
+            }
```

Comment:
as discussed with @mhsdesign  this has to be moved to the CR, otherwise hooks will be retriggered during publishing

Reply:
resolved with a3aa2266bb5d77fa55b64e6a76ca09aeb18c6e14